cypress/e2e/admin/forcePasswordChangeTeacher.feature

```diff
@@ -0,0 +1,95 @@
+@regression_test
+@stable_test
+Feature: Force Password Change - user must set a new password during login
+
+    As a user, I have to set a new password during login
+
+    Scenario: Admin updates the password for the users, they set a new password during login.
+
+        # pre-condition: admin logs in to create their account in a school
+        Given I am logged in as a '<admin>' at '<namespace>'
+
+        # admin creates a new teacher
+        When I click on administration in menu
+        When I go to '<role_to_manage>' administration
+        When I click on FAB
+        When I click on Add User in opened FAB for '<role_to_manage>'
+        When I fill out the user creation form for '<user_firstname>' '<user_lastname>' with email '<user_email>'
+        When I click on add button to add '<role_to_manage>'
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        Then I can see the user with email '<user_email>' in the table
+
+        # admin generates the registeration link to proceed with teacher  registration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        Then I click on the button Generate Personal Registration Link for teacher
+
+        # teacher performs registration via the generated link with PIN
+        Then I visit to the generated registration link
+        When I choose the language for the registration process
+        Then I click on the button Next to proceed to next step
+        Then I click again on the button Next to proceed to the personal data information page
+        Then I see my first name '<user_firstname>'
+        Then I see my last name '<user_lastname>'
+        Then I set a new password on personal data page
+        When I click on the button Next to proceed to the next step
+        When I accept the privacy and terms of use consents
+        Then I click on the button Next to proceed to the registration pin step
+        When I request a new registration pin
+        Then I retrieve the registration pin to enter it into the form for '<namespace>'
+        Then I click on the button Send and Get Started to successfully complete the registration process
```

Comment:
I think here we need to rephrase the step as it seems here we have are performing two steps. Or am I understood it incorrect?